packages/react-component-library/src/components/Timeline/context/reducer.ts

```diff
@@ -3,7 +3,7 @@ import {
   eachMonthOfInterval,
   eachWeekOfInterval,
 } from 'date-fns'
-import { padStart } from 'lodash'
+import { padStart, isNumber } from 'lodash'
```

Comment:
Would be nice to sort `isNumber` and `padStart`:

```suggestion
import { isNumber, padStart } from 'lodash'
```